js/lightbeam.js

```diff
@@ -142,8 +142,12 @@ const lightbeam = {
   resetData() {
     const resetData = document.getElementById('reset-data-button');
     resetData.addEventListener('click', async () => {
-      await storeChild.reset();
-      window.location.reload();
+      const confirmation = confirm('Pressing OK will delete all'
+       + ' Lightbeam information. Are you sure?');
+      if (confirmation) {
+        await storeChild.reset();
+        window.location.reload();
+      }
```

Comment:
@joanasouza as per @jonathanKingston 's we could do it like -
`messagePart1` & `messagePart2`?
confirm(`${m1} ${m2}`);